config.toml

```diff
@@ -1,35 +1,22 @@
-baseURL = "http://datatogether.org"
-relativeURLs = true
-languageCode = "en-us"
+baseURL = "https://datatogether.org"
```

Comment:
This https switch _might_ cause some issues on builds, but I think it's worth a shot. Let's keep this change & if things explode we can revert